integration-tests/src/main.ts

```diff
@@ -267,7 +267,10 @@ async function run(): Promise<void> {
         )} 🛠️ Creating incident in Pagerduty (if applicable)`,
         async () => {
           await createPagerdutyIncident(testsFolder, {
-            service: core.getInput('module_id'),
+            service:
+              core.getInput('incident_service') !== ''
+                ? core.getInput('incident_service')
+                : core.getInput('module_id'),
```

Comment:
Looks good, but can be shortened to following if we want:
 
```suggestion
            service: core.getInput('incident_service') || core.getInput('module_id'),
```